platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -128,17 +130,24 @@
     }
 
     /**
-     * Interface for a callback to received additional error information in a {@link SceneUpdateError}
-     * Triggered after a call of {@link #applySceneUpdates()} or {@link #loadSceneFile(String, List<SceneUpdate>)}
-     * Listener should be set with {@link #setSceneUpdateErrorListener(SceneUpdateErrorListener)}
-     * The callback will be run on the main (UI) thread.
+     * Interface for listening to scene load status information.
+     * Triggered after a call of {@link #updateScene(List<SceneUpdate>)} or {@link #loadSceneFile(String, List<SceneUpdate>)}
+     * Listener should be set with {@link #setSceneLoadListener(SceneLoadListener)}
+     * The callbacks will be run on the main (UI) thread.
      */
-    public interface SceneUpdateErrorListener {
+    public interface SceneLoadListener {
+        /**
+         * Received when a scene load succeeded.
+         * @param sceneId returned by {@link #updateScene(List<SceneUpdate>)} or {@link #loadSceneFile(String, List<SceneUpdate>)}
+         */
+        void onSceneReady(int sceneId);
+
         /**
-         * Receive error status when a scene update failed
+         * Receive error status when a scene load failed
+         * @param sceneId returned by {@link #updateScene(List<SceneUpdate>)} or {@link #loadSceneFile(String, List<SceneUpdate>)}
          * @param sceneUpdateError The  {@link SceneUpdateError} holding error informations
          */
-        void onSceneUpdateError(SceneUpdateError sceneUpdateError);
+        void onSceneError(int sceneId, SceneUpdateError sceneUpdateError);
```

Comment:
Correct me if I misunderstood, I think we decided (last week?) to club these 2 methods in a single one and let user handle the functionality based on the error state.